### PR TITLE
Initial support for torimochi widget ads via amp-ad plugin.

### PR DESCRIPTION
**Pull request by yuuki-tanaka**
Initial support for torimochi widget ads via amp-ad plugin.

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -941,6 +941,8 @@ export const adConfig = {
     consentHandlingOverride: true,
   },
 
+  'torimochi': {},
```
Comment (torch2424) [inline ads/_config.js]: So I noticed here you are not implementing `renderStartImplemented`. If possible, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

Let me know what you think. Thank you! 😄

File: ads/torimochi.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/torimochi.js]: Incorrect Copyright year 😄

File: ads/torimochi.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/torimochi.md]: Incorrect copyright year 😄

File: ads/torimochi.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+</amp-ad>
+```
+
+## Configuration
+
+For details on the configuration semantics, please contact the ad network or refer to their documentation. 
+
+Supported parameters:
+
+- `data-adtype`
```
Comment (torch2424) [inline ads/torimochi.md]: Thanks for adding your documentation. For the supplied parameters, could you specify which are optional vs. required? [Here is an example](https://github.com/ampproject/amphtml/blob/master/ads/_ping_.md ). Thank you! 😄

**Comment by yuuki-tanaka** [comment]
@torch2424 
I am sorry to be late.
I fixed the pointed out items, so please review.

File: ads/_config.js [hunk 5]
```diff
@@ -941,6 +941,10 @@ export const adConfig = {
     consentHandlingOverride: true,
   },
 
+  'torimochi': {
+    renderStartImplemented: true,
```
Comment (torch2424) [inline ads/_config.js]: Hello! 

So everything looks good, and everything is working:

screen shot 2019-02-25 at 7 15 33 pm

However, it seems like the `renderStart` is not working. I set some logging for `renderStart_()` in `amp-ad-xorigin-iframe-handler.js`. But it never seems to get called. 

Does your vendor implement `renderStart`? If not, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

Let me know what you think. Thank you! 😄

**Comment by yuuki-tanaka** [comment]
@torch2424 
I'm sorry. I do not understand ```renderStart``` much. 
However, I tried implementing ```renderStart```, but what about?

**Comment by torch2424** [comment]
@yuuki-tanaka Looks like your last commit worked!

screen shot 2019-02-26 at 12 50 58 pm

Thank you very much for your contribution! 😄 

Merging...

**Pull request merged**